### ISSUE TITLE
Need new data download link

### ISSUE THREAD

**Issue by Alllcher** [mmbannert/rigid-v-nonrigid-flow#8]
Link is expired.
Thank you very much for your time and consideration. I look forward to hearing from you soon.

**Comment by mmbannert** [comment]
Hi! The purpose of this repo was simply to visualize a problem that we had with Kubric. Kubric is a very useful data generating pipeline but its documentation a little incomplete sometimes. So I created this repo to communicate the problem that we were facing to other scientists and to make it as easy as possible for them to reproduce the problem for themselves. 

We have solved the problem in the meantime. I just never got around to updating the README. I just updated the plots for Kubric; as you can see, they look fine now for Kubric as well. Sorry about the confusion!

If you are still interested in the data, I made them available [here](https://owncloud.tuebingen.mpg.de/index.php/s/T4KZ2eoLjbgDrSm) for a few more days.

**Comment by Alllcher** [comment]
Thank you for your reply. I have another question, can I use /tools/flowcalc.py to calculate the rigid flow through the depth camera? I can obtain depth information and the camera's intrinsic and extrinsic matrices for this camera.

**Comment by mmbannert** [comment]
Yes, I think that you can. Take a look also at the very useful book by Szeliski, which I referenced in the code.
